### ISSUE TITLE
Beast Master Achievement, Disappearing Pets

### ISSUE THREAD

**Issue by eatmoreyogurt** [HabitRPG/habitica#1818]
I just got the Beast Master achievement... thing is that I only have 15 pets, so I definitely shouldn't have it yet.

My user ID is 9641c638-a0b1-4251-94cc-9598a1323f56, and I'm using Chrome as my browser.

**Comment by octern** [comment]
Just happened to me as well. It came up upon opening the page. User ID 9b508a2e-1d1b-44ee-a437-266583e756e3

**Comment by lefnire** [comment]
All you need to do is refresh the page.

fixed in https://github.com/HabitRPG/habitrpg/commit/224cbe2277a774c6e571183b6d1e90ec9c2cc62b , rolled out change. Beatsmaster achievement will be removed the next time your user is saved (from a + habit or whatever).

**Comment by lefnire** [comment]
Man, the dupes are coming in strong... :/ can't wait for tomorrow when everyone will have refreshed their pages anyway (crossing my fingers).

Some background info: I pushed UserSchema migration which lays the groundwork for mounts. It overhauls a lot of pets and items stuff (was a long-time coming), and we needed to roll out now so we could start writing translations to `items.coffee`.

The update requires that users refresh the page. I tweeted, facebook'd, tavern'd, tumblr'd - but alas, some stragglers aren't refreshin'. It's benign - beastMaster won't go through, they still have their pets - just gotta refresh
